src/cpx_callbacks.jl

```diff
@@ -19,8 +19,8 @@ function setcallbackcut(cbdata::CallbackData, where::Cint, ind::Vector{Cint}, va
     @assert length(val) == len
     sns = convert(Cint, sense)
     stat = @cpx_ccall(cutcallbackadd, Cint, (
-                      Ptr{Void},
-                      Ptr{Void},
+                      Ptr{Nothing},
```

Comment:
Please use `Cvoid` instead of `Nothing` for `void*` types.

Reply:
fixed in last commit